app/models/concerns/shared_parent_methods.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SharedParentMethods
+  # class methods for Exercise/Pain/BodyPart models
+  def by_name
+    order('lower(name) ASC')
+  end
+
+  def search(terms)
+    if terms.blank?
+      all
+    else
+      where('name ILIKE ?', "%#{terms}%")
+    end
+  end
+end
```

Comment:
@kelseyhuse30 I'm curious about what you think about this approach. Since I had 3 models that share the same methods, I pulled them out into this shared one. But I had a hard time naming it because the 3 models have nothing in common other that being base-level parent relationship for other models. When naming is this hard, it feels like a code smell to me. Have you encountered this approach before?

Reply:
What about naming the concern something like 'Searchable' ?

Reply:
I like the sound of `Searchable`. Though I'm thinking the `name` attribute is more of its unifying idea. Maybe I'll name it something like `Naming` or `Nameable`